l10n_fr_hr/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 GRAO
```

Comment:
GRAO ;-)

Reply:
I fork GRAP to GRAO didn't I tell you ? :stuck_out_tongue: